main.yml

```diff
@@ -15,7 +15,7 @@
 
     - name: Ensure pacman cache is up to date
       community.general.pacman:
-        update_cache: yes
+        update_cache: "yes"
```

Comment:
I use truthy, `true` in this case.

Reply:
Actually the reason why I used double quotes was the lint. It was giving me a warning otherwise. 

Seems that default truthy values are only `true` and `false` (from the Truthy docs):

> truthy: Use this rule to forbid non-explictly typed truthy values other than allowed ones (by default: true and false), for example YES or off.

No problem. Then we need to update truthy configuration to accept yes and no values. I will fix it and amend the changes.